File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	
	public List<TbBrand> findAll();

	public PageResult<TbBrand> findPage(int page, int rows);
	
	//public PageResult<TbBrand> findPage(int page, int rows, TbBrand tbBrand);
	
	public void add(TbBrand tbBrand);
	
	public void update(TbBrand tbBrand);
	
	public TbBrand findOne(Long id);
	
	public void delete(Long[] ids);
	
	public List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function($http){
    		this.findAll =  function(){
    			return $http.get('../brand/findAll.do');
    		};
    		
    		this.findPage = function(page,rows){
    			return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    		};
    		
    		this.save = function(entity){
    			var methodName = 'add';
    			if (entity.id != null) {
    				methodName = 'update'
    			};
    			return $http.post('../brand/'+methodName+'.do', entity);
    		}
    		
    		this.findOne = function(id){
    			return $http.get('../brand/findOne.do?id='+id);
    		}
    		
    		this.update = function(entity){
    			return $http.post('../brand/update.do', entity);
    		}
    		
    		this.dele = function(ids){
    			return $http.get('../brand/delete.do?ids='+ids);
    		}
    		
    		this.selectOptionList = function(){
    			return $http.get('../brand/selectOptionList.do');
    		}
});